apps/simulators/src/assets/config.json

```diff
@@ -2,8 +2,8 @@
   "appId": "simulators",
   "appName": "BioSimulators",
   "appNameParts": ["Bio", "Simulators"],
-  "logo": "assets/images/biosimulators-logo/logo.svg",
-  "banner": "assets/images/biosimulators-logo/logo-white.svg",
+  "logo": "assets/images/biosimulators-logo/logo-white-edited.png",
+  "banner": "assets/images/biosimulators-logo/logo-white-fotor-2023080163322.png",
```

Comment:
I don't see `logo-white-fotor-2023080163322.png` in the repo or your new commits.

Reply:
You know what, you're absolutely right about that! This is a `.png` that I edited online and cleaned up the filename. It would probably help if I called the correct filepath in `config`...hehe.